### PR TITLE
Bikesheddy Mime P.R.

### PR DESCRIPTION
**Pull request by pireamaineach**
New Book:
Principles

## About The Pull Request

Don't get me wrong, the mime powers P.R. was a blessing but the current book is just... ugh.

## Why It's Good For The Game

New Book!

## Changelog
:cl:
tweak: ~~New book.~~ Just modifies the text for the "Guide to Dank Mimery" because it sounds a little stupid.
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/mime.dm [hunk 1]
```diff
@@ -49,19 +49,27 @@
...
+	var/dat = "<B>Principles of Mimery</B><BR>"
+	dat += "This book teaches mastery over one of three classic mime routines.<BR>"
+	dat += "Once you've mastered your routine, the book, too, becomes invisible.<BR>"
 	dat += "<HR>"
-	dat += "<A href='byond://?src=[REF(src)];invisible_wall=1'>Invisible Wall</A><BR>"
-	dat += "<A href='byond://?src=[REF(src)];invisible_chair=1'>Invisible Chair</A><BR>"
-	dat += "<A href='byond://?src=[REF(src)];invisible_box=1'>Invisible Box</A><BR>"
+	dat += "<B>Table of Contents</B><BR>"
+	dat += "<BR>"
+	dat += "Indroduction 3<BR>"
```
Comment (nfreader) [inline code/modules/jobs/job_types/mime.dm]: Indoruciton

File: code/modules/jobs/job_types/mime.dm [hunk 2]
```diff
@@ -49,19 +49,27 @@
...
+	dat += "Summary 12<BR>"
+	dat += "A History of Mime 19<BR>"
+	dat += "Mime Time Line 24<BR>"
```
Comment (nfreader) [inline code/modules/jobs/job_types/mime.dm]: Timeline is one word

File: code/modules/jobs/job_types/mime.dm (hunk 1 above)
Comment (pireamaineach) [reply code/modules/jobs/job_types/mime.dm]: ebic

File: code/modules/jobs/job_types/mime.dm (hunk 2 above)
Comment (pireamaineach) [reply code/modules/jobs/job_types/mime.dm]: Actually in this context it's Time Line.

**Comment by bgobandit** [comment]
why get rid of the dank mimery pun though

**Comment by pireamaineach** [comment]
> why get rid of the dank mimery pun though
> 
> also
> 
> > criticizes writing
> > writes a better book with the correct English of "Indroduction"

Because Dank is a stupid word.

**Comment by bgobandit** [comment]
it's a reference to dank memes

**Comment by pireamaineach** [comment]
Which is a shit joke, sorry mate. Nothing becomes outdated and cringey faster than slang.

**Comment by bgobandit** [comment]
this server is built on slang and references, if you don't like them then good luck censoring them all out of our code

**Comment by wesoda25** [comment]
Unless you actually write information for those sections this is a lame PR

**Comment by ishitbyabullet** [comment]
well boys, i guess slang is dead. if anyone uses a slang word ever again then they will be **terminated**.

In all seriousness, if you really think dank mimery is only a joke that window lickers will ever like then you should probably adjust the changelog to say "the guide to dank mimery has sold out" while saying "a new mime book, [insert title here], has been published"

**Comment by wesoda25** [comment]
no guide to dank mimery is a stupid name and principles of mimery is much better

**Comment by bgobandit** [comment]
the fake page sections are also, while I can see the idea of them, unnecessary and potentially confusing, because I guarantee at least one new player will be like "wtf why are the other links disabled"

**Comment by pireamaineach** [comment]
yes pls give me the indroduction power

y indroduction not work pls

**Comment by bgobandit** [comment]
you haven't seen the adminhelps we get on a daily basis

**Comment by GuyonBroadway** [comment]
"The Profressionals Comprehensive guide to Performing Mime arts vol2: on a history of mimed and furniture"

**Comment by HomocidlePancakes** [comment]
Why does The Mime Timeline takes up only 6 pages?

**Comment by pireamaineach** [comment]
Because 6 rhymes with sex and that's EBIC!

**Comment by bgobandit** [comment]
>"dank memes" is a shitty joke"

> lol 6 rhymes with sex le epik meme

File: code/modules/jobs/job_types/mime.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/jobs/job_types/mime.dm]: what

**Comment by Detective-Google** [comment]
>indroduction

**Comment by ExcessiveUseOfCobblestone** [comment]
This is dumb, it should really just be an alert or something.

**Comment by pireamaineach** [comment]
> > indroduction
> 
> Good grammar fix 10/10

Already fixed.

**Comment by pireamaineach** [comment]
> This is dumb, it should really just be an alert or something.

I mean, maybe, but it's not exactly my fault the book exists.

File: code/modules/jobs/job_types/mime.dm (hunk 2 above)
Comment (pireamaineach) [reply code/modules/jobs/job_types/mime.dm]: https://www.merriam-webster.com/dictionary/time%20line

**Comment by SpinnerMaster** [comment]
Yo what if the mime book was written entirely in French except for the powers part.

**Comment by ExcessiveUseOfCobblestone** [comment]
> > This is dumb, it should really just be an alert or something.
> 
> I mean, maybe, but it's not exactly my fault the book exists.

???

This in "This is dumb" is the menu. I don't care it's fluffed as a book, that doesn't mean it needs a wonky ui for something that suffices with a input() `M/N: previously said alert but you wouldn't have a 4th cancel option`

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
Formalizing ^^

Just make it an input, no need for href bafoonery. No other spell granter does this.

In actuality it should prob be a granter subtype but OOS for this PR imo

**Comment by pireamaineach** [comment]
I mean, this's just a formatting P.R. 
If you want that, then I'll see about making another P.R. but I'm not going to do a 180° and completely rework the system within this P.R.

**Comment by ExcessiveUseOfCobblestone** [comment]
kk I don't plan on painting the bike shed a new color.

**Comment by pireamaineach** [comment]
It's literally a book called "Guide to Dank Mimery."
It's one of the stupidest jokes that have weaseled their way into the game.
@ExcessiveUseOfCobblestone

**Comment by ExcessiveUseOfCobblestone** [comment]
Yes I looked at the PR contents. It's really inconsequential that a book you use once roundstart that gets thrown away (if not deleted) afterwards is titled "Dank Mimery".

**Comment by pireamaineach** [comment]
You're not wrong but if we can change the book to not make people wince every time they look at it, why wouldn't we?

**Comment by ExcessiveUseOfCobblestone** [comment]
Because, since mime players are actually masochists, they enjoy it.

**Comment by pireamaineach** [comment]
I'm a mime player.

Cringe is not an enjoyable form of pain.

**Comment by ExcessiveUseOfCobblestone** [comment]
something something the non-vocal majority

**Comment by pireamaineach** [comment]
Yes because "Guide to Dank Mimery" isn't going to make 99/100 people want to kill themselves in the year of our lord two-thousand and nineteen.

**Comment by ExcessiveUseOfCobblestone** [comment]
I was making a mime joke

**Pull request closed without merging**